TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php

```diff
@@ -267,7 +273,7 @@ protected function parseElement(\XMLReader $xmlReader)
                 $this->nodeIdentifierStack[] = $xmlReader->getAttribute('identifier');
                 // update current path
                 $nodeName = $xmlReader->getAttribute('nodeName');
-                if ($nodeName !== '/') {
+                if ($nodeName !== '/' && $nodeName !== '') {
```

Comment:
Ok, no, this is indeed correct. The test `\TYPO3\TYPO3CR\Tests\Functional\Domain\NodeDataExportServiceTest::aSingleNodeExportedWithNodeDataExportCanBeImportedWithNodeDataImport` exports the root node and a single sub node. In this scenario the root node will have an empty node name (and not as the current code would suggest "/"), which basically is the root node... Technically we cannot correctly import it if you look at the test. So this behavior seems fine to me. I guess we should rethink that overall though...